### PR TITLE
improve log formatting

### PR DESCRIPTION
**Pull request by arvidn**

### CODE REVIEW

**Review by hoffmang9 — DISMISSED** [review]
lgtm

File: chia/full_node/mempool_manager.py [hunk 1]
```diff
@@ -428,10 +432,13 @@ async def add_spendbundle(
 
         new_item = MempoolItem(new_spend, uint64(fees), npc_result, cost, spend_name, additions, removals, program)
         self.mempool.add_to_pool(new_item)
-        log.info(
-            f"add_spendbundle took {time.time() - start_time} seconds, cost {cost} "
-            f"({round(100.0 * cost/self.constants.MAX_BLOCK_COST_CLVM, 3)}%)"
+        now = time.time()
+        log.log(
+            logging.WARNING if now - start_time > 1 else logging.DEBUG,
+            f"add_spendbundle {spend_name.hex()} took {now - start_time:0.2f} seconds. "
```
Comment (emlowe) [inline chia/full_node/mempool_manager.py:438]: Assuming this is the same spend_name used everwhere , this is normally logged as just {spend_name} and not {spend_name.hex()} - I believe it's a hash so maybe hex() is better but should it be consistent in the log?

Comment (arvidn) [reply chia/full_node/mempool_manager.py:438]: good point. In fact, in one of the places I just used `{spend_name}`. perhaps we have a custom formatter for `bytes32` to always print it as hex

Comment (arvidn) [reply chia/full_node/mempool_manager.py]: updated

**Review by emlowe — APPROVED** [review]
looks good

**Pull request merged**